mlflow/server/js/src/experiment-tracking/components/CompareRunBox.js

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Row, Col, Select } from 'antd';
+import { Typography } from '@databricks/design-system';
+import { RunInfo } from '../sdk/MlflowMessages';
+import { LazyPlot } from './LazyPlot';
+import CompareRunUtil from './CompareRunUtil';
+
+const { Option, OptGroup } = Select;
+
+export const CompareRunBox = ({ runUuids, runInfos, metricLists, paramLists }) => {
+
+  const parameter = CompareRunUtil.getKeys(paramLists, true)[0];
+  const metric = CompareRunUtil.getKeys(metricLists, true)[0];
+  console.log(parameter);
+  console.log(metric);
+
+  const [xAxis, setXAxis] = useState({ key: parameter, isParam: true });
+  const [yAxis, setYAxis] = useState({ key: metric, isParam: true });
```

Comment:
```suggestion
  const [yAxis, setYAxis] = useState({ key: metric, isParam: false });
```

Reply:
@harupy 

If we need to pass metric as a value, isParm has to be true or else React will throw the following error:
```
SectionErrorBoundary.js:15 ReferenceError: mlflow is not defined
    at CompareRunBox (CompareRunBox.js:13:1)
    at renderWithHooks (react-dom.development.js:14985:1)
    at mountIndeterminateComponent (react-dom.development.js:17811:1)
    at beginWork (react-dom.development.js:19049:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:3945:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3994:1)
    at invokeGuardedCallback (react-dom.development.js:4056:1)
    at beginWork$1 (react-dom.development.js:23964:1)
    at performUnitOfWork (react-dom.development.js:22776:1)
    at workLoopSync (react-dom.development.js:22707:1) {componentStack: '\n    at CompareRunBox (http://localhost:3000/stati…/static/js/bundle.js:629561:43)\n    at MLFlowRoot'}
```

Reply:
![screen_shot 2022-07-26 at 0 19 37](https://user-images.githubusercontent.com/51813538/180814665-84e2a295-aa5c-421d-9d3a-437a38e7b988.png)

Reply:
Hey harupy I managed to fix it. Sorry it was a bug on my end.